tests/test_module.py

```diff
@@ -129,11 +129,14 @@ def _insert_data(redis, key, start_ts, samples_count, value):
             else:
                 assert actual_result
 
-    def _insert_agg_data(self, redis, key, agg_type, chunk_type="", fromTS=10, toTS=50):
+    def _insert_agg_data(self, redis, key, agg_type, chunk_type="", fromTS=10, toTS=50, key_create_args=None):
         agg_key = '%s_agg_%s_10' % (key, agg_type)
 
-        assert redis.execute_command('TS.CREATE', key, chunk_type)
-        assert redis.execute_command('TS.CREATE', agg_key, chunk_type)
+        if key_create_args is None:
```

Comment:
@danni-m so why not set the default value of `key_create_args` to `[]`?

Reply:
@gkorland because in python it means its a mutable "static" var:
https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments